### ISSUE TITLE
[Terra-Alert] Jaws screenreader doesn't read the alerts in windows

### ISSUE THREAD

**Issue by ln054300** [cerner/terra-core#2832]
# Issue Description

Jaws screen reader is unable to read the terra-alerts in windows whereas it works with voiceover in mac.  

## Steps to Reproduce
Install Jaws screen reader
Launch a web app that uses terra-alert component in a windows environment.
Screen read the page where there is an alert.

## Expected Behavior
The screen reader should be able to read the alert when an alert is popped up on the page.

## Environment

* Component Name and Version: Terra-alert ^3.12.0
* Browser Name and Version: Google Chrome 80.0.3987.85
* Node/npm Version: Node 8
* Webpack Version: 3.11.0
* Operating System and version (desktop or mobile): Windows 10

**Comment by ryanthemanuel** [comment]
@scottwilmarth how should alerts work with screenreaders.  Both in the case that the alert is present on initial load and when one gets added after the fact?  Do all alert types need the same treatment?

**Comment by scottwilmarth** [comment]
@ryanthemanuel, yes, alerts must work with screen readers. Alerts that are on the page with the initial load should be treated as polite. If they are set as assertive, they will more than likely interrupt the initial load description a screen reader would provide. As for the alerts that pop after the initial load, those should be based on the severity of the issue. All severe alerts should be set as assertive which would interrupt the current screen reader activity and immediately provide the details to the user. Other alerts with lesser severity should be set as polite. This will allow the screen reader to finish the current user activity before the alert is read.

**Comment by ryanthemanuel** [comment]
@scottwilmarth @saket2403 in order to accomplish everything in @scottwilmarth 's comment we will need to enhance how we handle alerts quite a bit (to have some kind of manager).  This is something we're planning on doing as part of the Orion/MPage Integration project.  For this issue, however, i'd like to see if we can align how the voice over technologies work (and then get all the bells and whistles with the enhancement we've got coming).
